tsimcne/tsimcne.py

```diff
@@ -471,7 +471,7 @@ def fit(self, X: torch.utils.data.Dataset | str | Path):
         train_dl = self.make_dataloader(X, True, self.data_transform)
 
         self.data_transform_none = get_transforms_unnormalized(
-            size=self.image_size, setting="none", use_ffcv=self.use_ffcv
+            size=self.image_size, setting="none", use_ffcv=self.use_ffcv # TODO: interferes with data_transforms usage - same as running get_transform for test?
```

Comment:
This can probably be replaced by the call to the new function `get_data_transform`